apps/ledger-live-mobile/src/components/Web3AppWebview/liveSDKLogic.test.ts

```diff
@@ -21,23 +21,15 @@ describe("prepareSignTransaction", () => {
     // Given
     const parentAccount = createAccount("12");
     const childAccount = createTokenAccount("22", "js:2:ethereum:0x012:");
-    const expectedResult: EvmTransaction = {
+    const expectedResult: Partial<EvmTransaction> = {
```

Comment:
Not sure that testing half of the expected result is enough.
The function is expected to return a full `Transaction` object, not a Partial one.

Reply:
This is what the prepareSignTransaction returns